src/NetAnalyzers/UnitTests/Microsoft.NetFramework.Analyzers/DoNotUseInsecureXSLTScriptExecutionXslCompiledTransformLoadInsecureInputSettingsTests.cs

```diff
@@ -798,51 +798,6 @@ private static void TestMethod(XsltSettings settings, XmlSecureResolver resolver
 Imports System.Xml
 Imports System.Xml.Xsl
 
-Namespace TestNamespace
-    Class TestClass
-        Private Shared Sub TestMethod(settings As XsltSettings, resolver As XmlSecureResolver)
-            Try
-            Catch
-                Throw
-            Finally
-                Dim xslCompiledTransform As New XslCompiledTransform()
-                settings.EnableScript = True
-                xslCompiledTransform.Load("""", settings, resolver)
-            End Try
-        End Sub
-    End Class
-End Namespace");
-        }
-
-        [Fact]
-        public async Task UseXslCompiledTransformLoadInputSettingsSetOneToTrueAndSecureResolverAsyncAwaitShouldNotGenerateDiagnostic()
```

Comment:
Duplicates of `UseXslCompiledTransformLoadInputSettingsSetOneToTrueAndNonSecureResolverInFinallyBlockShouldGenerateDiagnostic`.